spec/api-browser-window-spec.js

```diff
@@ -1396,6 +1396,68 @@ describe('BrowserWindow module', () => {
       })
     })
 
+    describe('"enableRemoteModule" option', () => {
+      it('enables the remote module by default', async () => {
+        const preload = path.join(fixtures, 'module', 'preload-remote.js')
+        w.destroy()
+        w = new BrowserWindow({
```

Comment:
Can you please use `openTheWindow()` to create a new BrowserWindow?

```js
const w = await openTheWindow(...)
```

Reply:
done